StripeFinancialConnections/StripeFinancialConnections/Source/Native/LinkAccountPicker/LinkAccountPickerViewController.swift

```diff
@@ -217,47 +213,62 @@ final class LinkAccountPickerViewController: UIViewController {
         self.footerView = footerView
         footerContainerView.addAndPinSubview(footerView)
 
-        bodyView.selectAccount(nil) // activate the logic to list all accounts
+        bodyView.selectAccounts([]) // activate the logic to list all accounts
     }
 
-    private func didSelectConectAccount() {
-        guard let selectedAccountTuple = dataSource.selectedAccountTuple else {
-            assertionFailure("user shouldn't be able to press the connect account button without an account")
+    private func didSelectConnectAccounts() {
```

Comment:
[equivalent web code for the function](https://git.corp.stripe.com/stripe-internal/stripe-js-v3/blob/f87b7a8a/src/linkedAccounts/inner/components/panes/v3/LinkAccountPickerPane/containers/ReturningUser.tsx#L146-L176)